docs/source/developers/overview.rst

```diff
@@ -64,12 +64,19 @@ checklist for using ``git``:
     straightforward.
 
     Once you have an updated local copy, you can push to your remote repo.  Note, since your
-    remote repo still holds the old history, you would need to do a force push.
+    remote repo still holds the old history, you would need to do a force push.  Most pushes
+    should use ``--force-with-lease``
 
     .. code:: console
 
-       $ git push --force origin branch
+       $ git push --force-with-lease origin branch
+
+    If that fails, use a regular ``--force`` push.
```

Comment:
Instead of stating the `--force` alternative, I think it would be better to explain in one simple sentence why `--force-with-lease` is preferrable over `--force` (because that option is not very well-known).

Reply:
Thanks for the feedback. Needed two sentences. Hopefully ok. Not sure if a link to the [documentation](https://git-scm.com/docs/git-push) is needed.